src/patches/dns.js

```diff
@@ -18,6 +18,10 @@
 
     function dnsLookupWrapperResponse (error, address, family) {
       if (error && error.code === dns.NOTFOUND && --remaining > 0) {
+        // Using a logger other than the console would be ideal. Since this
+        // code is injected as a patch, it is hard to get access to a better
+        // logger
+        console.log(`DNS lookup of ${hostname} failed and will be retried ${remaining} more times`);
```

Comment:
While I can't think of any real examples where it would make a difference, `console.error` strikes me as slightly more appropriate since it would use `stderr` rather than `stdout`.

Reply:
Good point. I'll change